src/Algolia.Search/Models/Search/Query.cs

```diff
@@ -37,16 +37,23 @@ public class Query
         /// <summary>
         /// Create a new query with an empty search query
         /// </summary>
-        /// <param name="searchQuery"></param>
-        public Query(string searchQuery = null)
+        public Query()
         {
-            SearchQuery = searchQuery;
+        }
+
+        /// <summary>
+        /// Create a new query with an empty search query
+        /// </summary>
+        /// <param name="searchSearchQuery"></param>
+        public Query(string searchSearchQuery)
+        {
+            SearchQuery = searchSearchQuery;
```

Comment:
1. Can we keep the original `searchQuery` variable name instead of `searchSearchQuery` as it stutters
2. Are we still retaining the previous behavior which makes the value default to `null` if unspecified? If not, why do we want to change this?

Reply:
I don't remember why I made this change. I'll look into it.